qiskit_nature/results/utils/protein_plotter.py

```diff
@@ -0,0 +1,144 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2022.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+"""An auxiliary class that plots aminoacids of a molecule
+ in a ProteinFoldingResult."""
+import matplotlib.pyplot as plt
+import numpy as np
+import qiskit_nature.optionals as _optionals
+
+from qiskit_nature.results.utils.protein_shape_file_gen import ProteinShapeFileGen
+
+
+@_optionals.HAS_MATPLOTLIB.require_in_instance
+class ProteinPlotter:
+    """Plotter class for ProteinFoldingResult."""
+
+    def __init__(self, shape_gen: ProteinShapeFileGen) -> None:
+        """
+        Args:
+            protein_folding_result: The protein folding result to be plotted
+        """
+
+        self._shape_gen = shape_gen
+        # pylint: disable=unbalanced-tuple-unpacking
```

Comment:
What exactly are you ignoring here? I.e. why does this error occur in the first place and why is it okay to be ignored?